devices.js

```diff
@@ -195,6 +195,27 @@ const preset = {
             toZigbee: preset.light_onoff_brightness_colortemp_color({supportsHS: true, ...options}).toZigbee.concat([tz.ledvance_commands]),
         }),
     };
+    preset.tradfri = {
+        light_onoff_brightness: (options = {}) => ({
+            ...preset.light_onoff_brightness(options),
+            exposes: preset.light_onoff_brightness(options).exposes.concat([
+                exposes.enum('power_on_behavior', ea.ALL, ['off', 'previous', 'on'])
+                    .withDescription('Controls the behaviour when the device is powered on')]),
+        }),
+        light_onoff_brightness_colortemp: (options = {disableColorTempStartup: true, colorTempRange: [250, 454]}) => ({
+            ...preset.light_onoff_brightness_colortemp(options),
+            exposes: preset.light_onoff_brightness_colortemp(options).exposes.concat([
+                exposes.enum('power_on_behavior', ea.ALL, ['off', 'previous', 'on'])
+                    .withDescription('Controls the behaviour when the device is powered on')]),
+        }),
+        light_onoff_brightness_colortemp_color: (options = {disableEffect: true,
+            disableColorTempStartup: true, colorTempRange: [250, 454]}) => ({
+            ...preset.light_onoff_brightness_colortemp_color(options),
+            exposes: preset.light_onoff_brightness_colortemp_color(options).exposes.concat([
```

Comment:
Also add `onEvent: ikea.bulbOnEvent` here (above exposes) and remove `onEvent: ikea.bulbOnEvent` from all lights.

Reply:
Should I also move ota: to the presets?